crowbar_framework/config/routes.rb

```diff
@@ -274,6 +274,7 @@
       only: [:show, :update] do
```

Comment:
Looks like broken indentation here (although I know it's not related to this PR).